### ISSUE TITLE
after Scaffold Rising This issue please solve my problem

### ISSUE THREAD

**Issue by oneline** [bhrugen/Bulky#4]
Section 8.2 after Scaffold   Rising This issue please solve my problem

On Program.cs 
var app= builder.Build();
System.AggregateException 'Some services are not able to be contructed(Error while validating the service the sevice descriptor service Type:
DataAccess.Repository.IRepository.IUnitWork

builder (Microsoft.AspNetCore.Builder.WebApplicationBuilder)
![Error](https://user-images.githubusercontent.com/49000395/181796937-435abbca-17b0-4a4c-8896-95beb75ce188.jpg)

**Comment by Anandks1993** [comment]
I am also getting this same error. Did you @oneline resolve this issue?

**Comment by Anandks1993** [comment]
I found the solution in Udemy Q&A as one co-student fixed this issue. Will post the solution he posted here below.

`The scaffolding of Identity created a new class ApplicationDbContext.cs in my BulkyBookWeb project. This was conflicting with the existing ApplicationDbContext.cs in the BulkyBook.DataAccess class library. After I commented out the newly created ApplicationDbContext.cs, the application showed no error anymore.`

![2022-08-12_11-00-45-f73b9fb45da91c1dbf7f638df9b0b12c](https://user-images.githubusercontent.com/26414695/184805038-e19713f4-e88c-4605-ab23-51ddfb6f41b3.png)
